### ISSUE TITLE
Regression: LoadImage PyAV path decodes PNG mask as green RGB values instead of black/white

### ISSUE THREAD

**Issue by G7b9** [comfyanonymous/ComfyUI#14128]
### Custom Node Testing

- [x] I have tried disabling custom nodes and the issue persists (see [how to disable custom nodes](https://docs.comfy.org/troubleshooting/custom-node-issues#step-1%3A-test-with-all-custom-nodes-disabled) if you need help)

### Expected Behavior

`LoadImage` should decode static PNG mask images consistently with the previous Pillow-based implementation.

For a binary / grayscale / palette PNG mask, the `IMAGE` output should preserve the expected black/white RGB values after loading.

For example:

- background: RGB (0, 0, 0)
- foreground / mask area: RGB (255, 255, 255)

Then the following workflow should produce a clean 0/1 mask:

LoadImage.IMAGE -> Convert Image to Mask(channel="red")

The red channel of white mask areas should be 1.0, and the red channel of black background areas should be 0.0.
## workflow snapshot:

<img width="1433" height="888" alt="Image" src="https://github.com/user-attachments/assets/835cf38d-7a6c-4c45-aea9-ce9e1aefecae" />

### Actual Behavior

After updating ComfyUI, the same PNG mask image is decoded into unexpected greenish RGB values when loaded with `LoadImage`.

When connecting `LoadImage.IMAGE` directly to `PreviewImage`, the mask is displayed with these colors:

- background: RGB (0, 134, 0)
- foreground / mask area: RGB (75, 255, 28)

This also affects `Convert Image to Mask(channel="red")`, because the foreground red channel becomes:

75 / 255 ≈ 0.294

instead of the expected value 1.0.

As a result, mask workflows that previously worked now produce incorrect mask values.

### Steps to Reproduce

1. Prepare a binary / grayscale / palette PNG mask image.
2. Load it with the built-in `LoadImage` node.
3. Connect `LoadImage.IMAGE` directly to `PreviewImage`.
4. Observe that the preview shows greenish colors instead of the expected black/white mask.
5. Connect the workflow:

   LoadImage.IMAGE -> Convert Image to Mask(channel="red") -> Mask Preview

6. Observe that the mask value is not a clean 0/1 mask. The foreground red value is around 75/255 ≈ 0.294 instead of 1.0.

This worked correctly with the previous Pillow-based `LoadImage` implementation.

### Debug Logs

```powershell
There is no Python exception or traceback. The issue is a decoded pixel value regression.

The relevant behavior difference appears to be in `LoadImage`.

Previous implementation used Pillow directly:


img = Image.open(image_path)
image = i.convert("RGB")
image = np.array(image).astype(np.float32) / 255.0

Current implementation first tries the VideoFromFile path:

components = InputImpl.VideoFromFile(image_path).get_components()
if components.images.shape[0] > 0:
    return components.images, ...

With the current implementation, the same PNG mask is decoded as:

background: (0, 134, 0)
foreground: (75, 255, 28)

Expected decoded RGB values should be black/white, for example:

background: (0, 0, 0)
foreground: (255, 255, 255)

Please let me know if a sample image is needed. I can provide the PNG mask file that reproduces the issue.
```

**Comment by comfyanonymous** [comment]
can you provide an example image with the issue?

**Comment by G7b9** [comment]
@comfyanonymous 
<img width="1438" height="1094" alt="Image" src="https://github.com/user-attachments/assets/4b7a679f-0485-46a6-8dfa-03eeb42bb295" />
Furthermore, I modified the code to the following version, and it returned to normal.：

```python
class LoadImage:
    @classmethod
    def INPUT_TYPES(s):
        input_dir = folder_paths.get_input_directory()
        files = [f for f in os.listdir(input_dir) if os.path.isfile(os.path.join(input_dir, f))]
        files = folder_paths.filter_files_content_types(files, ["image"])
        return {"required":
                    {"image": (sorted(files), {"image_upload": True})},
                }

    CATEGORY = "image"
    SEARCH_ALIASES = ["load image", "open image", "import image", "image input", "upload image", "read image", "image loader"]

    RETURN_TYPES = ("IMAGE", "MASK")
    FUNCTION = "load_image"
    def load_image(self, image):
        image_path = folder_paths.get_annotated_filepath(image)

        img = node_helpers.pillow(Image.open, image_path)

        output_images = []
        output_masks = []
        w, h = None, None

        for i in ImageSequence.Iterator(img):
            i = node_helpers.pillow(ImageOps.exif_transpose, i)

            if i.mode == 'I':
                i = i.point(lambda i: i * (1 / 255))
            image = i.convert("RGB")

            if len(output_images) == 0:
                w = image.size[0]
                h = image.size[1]

            if image.size[0] != w or image.size[1] != h:
                continue

            image = np.array(image).astype(np.float32) / 255.0
            image = torch.from_numpy(image)[None,]
            if 'A' in i.getbands():
                mask = np.array(i.getchannel('A')).astype(np.float32) / 255.0
                mask = 1. - torch.from_numpy(mask)
            elif i.mode == 'P' and 'transparency' in i.info:
                mask = np.array(i.convert('RGBA').getchannel('A')).astype(np.float32) / 255.0
                mask = 1. - torch.from_numpy(mask)
            else:
                mask = torch.zeros((64,64), dtype=torch.float32, device="cpu")
            output_images.append(image)
            output_masks.append(mask.unsqueeze(0))

            if img.format == "MPO":
                break  # ignore all frames except the first one for MPO format

        if len(output_images) > 1:
            output_image = torch.cat(output_images, dim=0)
            output_mask = torch.cat(output_masks, dim=0)
        else:
            output_image = output_images[0]
            output_mask = output_masks[0]

        return (output_image, output_mask)`
```

The buggy version：
```python
class LoadImage:
    @classmethod
    def INPUT_TYPES(s):
        input_dir = folder_paths.get_input_directory()
        files = [f for f in os.listdir(input_dir) if os.path.isfile(os.path.join(input_dir, f))]
        files = folder_paths.filter_files_content_types(files, ["image"])
        return {"required":
                    {"image": (sorted(files), {"image_upload": True})},
                }

    CATEGORY = "image"
    ESSENTIALS_CATEGORY = "Basics"
    SEARCH_ALIASES = ["load image", "open image", "import image", "image input", "upload image", "read image", "image loader"]

    RETURN_TYPES = ("IMAGE", "MASK")
    FUNCTION = "load_image"

    def load_image(self, image):
        image_path = folder_paths.get_annotated_filepath(image)

        dtype = comfy.model_management.intermediate_dtype()
        device = comfy.model_management.intermediate_device()

        components = InputImpl.VideoFromFile(image_path).get_components()
        if components.images.shape[0] > 0:
            return (components.images.to(device=device, dtype=dtype), (1.0 - components.alpha[..., -1]).to(device=device, dtype=dtype) if components.alpha is not None else torch.zeros((components.images.shape[0], 64, 64), dtype=dtype, device=device))

        # This code is left here to handle animated webp which pyav does not support loading
        img = node_helpers.pillow(Image.open, image_path)

        output_images = []
        output_masks = []
        w, h = None, None

        for i in ImageSequence.Iterator(img):
            i = node_helpers.pillow(ImageOps.exif_transpose, i)

            image = i.convert("RGB")

            if len(output_images) == 0:
                w = image.size[0]
                h = image.size[1]

            if image.size[0] != w or image.size[1] != h:
                continue

            image = np.array(image).astype(np.float32) / 255.0
            image = torch.from_numpy(image)[None,]
            if 'A' in i.getbands():
                mask = np.array(i.getchannel('A')).astype(np.float32) / 255.0
                mask = 1. - torch.from_numpy(mask)
            else:
                mask = torch.zeros((64, 64), dtype=torch.float32, device="cpu")
            output_images.append(image.to(dtype=dtype))
            output_masks.append(mask.unsqueeze(0).to(dtype=dtype))

        output_image = torch.cat(output_images, dim=0)
        output_mask = torch.cat(output_masks, dim=0)

        return (output_image.to(device=device, dtype=dtype), output_mask.to(device=device, dtype=dtype))
```

**Comment by comfyanonymous** [comment]
Should be fixed now: https://github.com/Comfy-Org/ComfyUI/commit/bcf805aaea6dd1935ceaa09d40700e2e93ce60c7
